packages/gatsby/src/utils/webpack/corejs-resolver.ts

```diff
@@ -0,0 +1,103 @@
+import Resolver from "enhanced-resolve/lib/Resolver"
+import * as path from "path"
```

Comment:
is the `* as` necessary?

Reply:
This is technically how you should do imports. It only works because we have moduleInterop enabled. I'm happy to do it with a regular import. I do feel we need a rule for it if we want to keep things consistent.

Reply:
Yeah we discussed early on to do the moduleInterop. We can add a rule to keep it consistent. It's not really the end of the world either way. And I don't have _strong_ opinions on it, but so far the industry has primarily used module interop, but maybe it'll change to more uses of `*`, but that's also why I'm trying to get us all to use named exports instead of deafult exports